src/Tests/dotnet.Tests/dotnet-new/DotnetModifyJsonPostActionTests.cs

```diff
@@ -0,0 +1,306 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Json.Nodes;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Tools.New.PostActionProcessors;
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Mocks;
+using Microsoft.TemplateEngine.TestHelper;
+using Moq;
+using Xunit;
+
+namespace Microsoft.DotNet.Cli.New.Tests
```

Comment:
A test with multiple changes to the json file would be very helpful!

Reply:
Hmm, in hindsight, I don't think running the PostAction twice in the same template will deliver the desired results.  Good idea to create a test for this; I'll see to that.

Reply:
I have added an additional end-to-end test which tests this.